Source/WebCore/platform/audio/PlatformMediaSessionManager.cpp

```diff
@@ -695,6 +700,27 @@ void PlatformMediaSessionManager::setOpusDecoderEnabled(bool enabled)
 #endif
 }
 
+void PlatformMediaSessionManager::setAlternateWebMPlayerEnabled(bool enabled)
+{
+#if ENABLE(ALTERNATE_WEBM_PLAYER)
+    if (m_alternateWebMPlayerEnabled != enabled) {
+        m_alternateWebMPlayerEnabled = enabled;
+        HTMLMediaElement::resetMediaEngines();
```

Comment:
So this will have to be done at the call site, or via a lambda set on `PlatformMediaSessionManager` at some point.